### PR TITLE
Add pagination to assets tag

### PR DESCRIPTION
**Pull request by edalzell**

### CODE REVIEW

**Comment by edalzell** [comment]
@jasonvarga this works but I'm having trouble getting tests in there. Any guidance would be appreciated.

**Comment by jasonvarga** [comment]
The asset object isn't enough. An actual file needs to exist. That's why you're doing Storage::fake(). Put a corresponding file in there.

**Comment by edalzell** [comment]
> The asset object isn't enough. An actual file needs to exist. That's why you're doing Storage::fake(). Put a corresponding file in there.

So the mock of the asset container doesn’t work?

File: tests/Tags/AssetsTagTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        return (string) Parse::template($tag, $data);
+    }
+
+    /** @test */
+    public function it_outputs_assets()
+    {
+        $container = (new AssetContainer)
+            ->handle('test_container')
+            ->disk('test');
```
Comment (jasonvarga) [inline tests/Tags/AssetsTagTest.php:38]: This isn't a mock, but this is what you're providing to AssetContainer::find() below.

**Pull request closed without merging**